src/main/ngapp/accio/src/app/editor/priority-queue.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { PriorityQueue } from './priority-queue';
+
+describe('PriorityQueue', () => {
```

Comment:
If you have time it would be good to add tests for the other functions as well.  If you don't have time them adding a TODO seems fine.

Reply:
I'll add a TODO comment for now, since we are pressed on time atm, and I'll make sure to come back to this if we're able to merge and integrate all our branches early!